spring-pulsar-cache-provider-caffeine/build.gradle

```diff
@@ -0,0 +1,59 @@
+plugins {
+	id 'org.springframework.pulsar.spring-module'
+	id 'com.github.johnrengelman.shadow' version '7.1.2'
+}
+
+description = 'Spring Pulsar Caffeine Cache Provider'
+
+dependencies {
+	api project (':spring-pulsar-cache-provider')
+	implementation 'com.github.ben-manes.caffeine:caffeine'
+	shadow project(':spring-pulsar-cache-provider')
+}
+
+jar {
+	archiveClassifier.set('original')
+}
+
+shadowJar {
+	archiveClassifier.set(null)
+	dependsOn(project.tasks.jar)
+	manifest {
+		inheritFrom project.tasks.jar.manifest
+	}
+	relocate 'com.github.benmanes.caffeine', 'org.springframework.pulsar.shade.com.github.benmanes.caffeine'
```

Comment:
The end result looks like this: 

<img width="569" alt="Screen Shot 2023-03-27 at 13 07 39" src="https://user-images.githubusercontent.com/28907971/228028663-6dba087e-cefa-4724-ab46-0179656b663c.png">